pages/docs/some-react-code/index.jsx

```diff
@@ -3,18 +3,18 @@ import Demo from './_Demo'
 import DocumentTitle from 'react-document-title'
 import { config } from 'config'
 
+exports.data = {
+  title: 'Some React Code',
+  date: `${new Date()}`
```

Comment:
( @rattrayalex the @ reply doesn't seem to auto fill in mobile and didn't want to butcher your name.)

I removed the date as it isn't needed. I did notice the version was pretty old, but that shouldn't matter with the ^. I did update it. That is the only other thing I can think to try. Everything that I submitted in the PR works on .28 with no issue that I can see.

@KyleAMathews did you want to keep those examples or no?